src/OS2UnicodeRanges.txt

```diff
@@ -0,0 +1,169 @@
+	Bopomofo Extended	31A0-31BF
+52	Hangul Compatibility Jamo	3130-318F
+53	Phags-pa	A840-A87F
+54	Enclosed CJK Letters And Months	3200-32FF
+55	CJK Compatibility	3300-33FF
+56	Hangul Syllables	AC00-D7AF
+57	Non-Plane 0	10000-10FFFF
+58	Phoenician	10900-1091F
+59	CJK Unified Ideographs	4E00-9FFF
+	CJK Radicals Supplement	2E80-2EFF
+	Kangxi Radicals	2F00-2FDF
+	Ideographic Description Characters	2FF0-2FFF
+	CJK Unified Ideographs Extension A	3400-4DBF
+	CJK Unified Ideographs Extension B	20000-2A6DF
+	Kanbun	3190-319F
+60	Private Use Area (plane 0)	E000-F8FF
+61	CJK Strokes	31C0-31EF
+	CJK Compatibility Ideographs	F900-FAFF
+	CJK Compatibility Ideographs Supplement	2F800-2FA1F
+62	Alphabetic Presentation Forms	FB00-FB4F
+63	Arabic Presentation Forms-A	FB50-FDFF
+64	Combining Half Marks	FE20-FE2F
+65	Vertical Forms	FE10-FE1F
+	CJK Compatibility Forms	FE30-FE4F
+66	Small Form Variants	FE50-FE6F
+67	Arabic Presentation Forms-B	FE70-FEFF
+68	Halfwidth And Fullwidth Forms	FF00-FFEF
+69	Specials	FFF0-FFFF
+70	Tibetan	0F00-0FFF
+71	Syriac	0700-074F
+72	Thaana	0780-07BF
+73	Sinhala	0D80-0DFF
+74	Myanmar	1000-109F
+75	Ethiopic	1200-137F
+	Ethiopic Supplement	1380-139F
+	Ethiopic Extended	2D80-2DDF
+76	Cherokee	13A0-13FF
+77	Unified Canadian Aboriginal Syllabics	1400-167F
+78	Ogham	1680-169F
+79	Runic	16A0-16FF
+80	Khmer	1780-17FF
+	Khmer Symbols	19E0-19FF
+81	Mongolian	1800-18AF
+82	Braille Patterns	2800-28FF
+83	Yi Syllables	A000-A48F
+	Yi Radicals	A490-A4CF
+84	Tagalog	1700-171F
+	Hanunoo	1720-173F
+	Buhid	1740-175F
+	Tagbanwa	1760-177F
+85	Old Italic	10300-1032F
+86	Gothic	10330-1034F
+87	Deseret	10400-1044F
+88	Byzantine Musical Symbols	1D000-1D0FF
+	Musical Symbols	1D100-1D1FF
+	Ancient Greek Musical Notation	1D200-1D24F
+89	Mathematical Alphanumeric Symbols	1D400-1D7FF
+90	Private Use (plane 15)	F0000-FFFFD
+	Private Use (plane 16)	100000-10FFFD
+91	Variation Selectors	FE00-FE0F
+	Variation Selectors Supplement	E0100-E01EF
+92	Tags	E0000-E007F
+93	Limbu	1900-194F
+94	Tai Le	1950-197F
+95	New Tai Lue	1980-19DF
+96	Buginese	1A00-1A1F
+97	Glagolitic	2C00-2C5F
+98	Tifinagh	2D30-2D7F
+99	Yijing Hexagram Symbols	4DC0-4DFF
+100	Syloti Nagri	A800-A82F
+101	Linear B Syllabary	10000-1007F
+	Linear B Ideograms	10080-100FF
+	Aegean Numbers	10100-1013F
+102	Ancient Greek Numbers	10140-1018F
+103	Ugaritic	10380-1039F
+104	Old Persian	103A0-103DF
+105	Shavian	10450-1047F
+106	Osmanya	10480-104AF
+107	Cypriot Syllabary	10800-1083F
+108	Kharoshthi	10A00-10A5F
+109	Tai Xuan Jing Symbols	1D300-1D35F
+110	Cuneiform	12000-123FF
+	Cuneiform Numbers and Punctuation	12400-1247F
+111	Counting Rod Numerals	1D360-1D37F
+112	Sundanese	1B80-1BBF
+113	Lepcha	1C00-1C4F
+114	Ol Chiki	1C50-1C7F
+115	Saurashtra	A880-A8DF
+116	Kayah Li	A900-A92F
+117	Rejang	A930-A95F
+118	Cham	AA00-AA5F
+119	Ancient Symbols	10190-101CF
+120	Phaistos Disc	101D0-101FF
+121	Carian	102A0-102DF
+	Lycian	10280-1029F
+	Lydian	10920-1093F
+122	Domino Tiles	1F030-1F09F
+	Mahjong Tiles	1F000-1F02F
```

Comment:
Do we really need to keep this file, I was under the impression that all bit are saturated and no new Unicode ranges are going to be added.

Reply:
The bits are saturated. Do they ever add ranges to assigned bits for new “Supplement” or “Extended” blocks?

Reply:
Apparently not, the list seems to not have been updated since OpenType 1.5 (released in 2008).

Reply:
Okay, I’ll revert this.